plugins/orchestrator/src/dataFormatters/DataFormatter.ts

```diff
@@ -0,0 +1,5 @@
+interface DataFormatter<Data, FormattedData> {
+  format(Data): FormattedData;
```

Comment:
```suggestion
  format(data: Data): FormattedData;
```
Otherwise the CI will fail.